### PR TITLE
Gnote: uses the same api as Tomboy, just changed the names.

### PR DESCRIPTION
**Pull request by thebopshoobop**

### CODE REVIEW

File: org.albert.extension.external.gnote.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        "name": "Gnote",
+        "version": "0.1",
+        "author": "Will Timpson",
+        "dependencies": ["gnote", "python-pydbus"],
+        "trigger": "gn "
+    }
+    print(json.dumps(metadata))
+    sys.exit(0)
+
+elif albert_op == "NAME":
```
Comment (ManuelSchneid3r) [inline org.albert.extension.external.gnote.py]: Same here (as in tomboy)

File: org.albert.extension.external.gnote.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    print("Gnote")
+    sys.exit(0)
+
+elif albert_op == "QUERY":
+    def build_action(name, interface, arguments=None):
+        action = {
+            "name": name,
+            "command": "dbus-send",
```
Comment (ManuelSchneid3r) [inline org.albert.extension.external.gnote.py]: looks like this is the old thingy. well if tomboy is done copy it again and change the names

File: org.albert.extension.external.gnote.py (hunk 1 above)
Comment (thebopshoobop) [inline org.albert.extension.external.gnote.py]: https://github.com/albertlauncher/external/pull/3/commits/0a1b698b52eeef6c4f1dbf38210329eb38255cd9

File: org.albert.extension.external.gnote.py (hunk 2 above)
Comment (thebopshoobop) [inline org.albert.extension.external.gnote.py]: https://github.com/albertlauncher/external/pull/3/commits/86157f6a0fa01b48d3e8e25d85f5d3f54bc0f7b0

**Pull request merged**